android/java/org/chromium/chrome/browser/ntp_background_images/RewardsBottomSheetDialogFragment.java

```diff
@@ -202,8 +202,8 @@ private void hideSponsoredImages() {
     }
 
     private void turnOnAds() {
+        BraveRewardsNativeWorker.getInstance().CreateRewardsWallet();
         BraveAdsNativeHelper.nativeSetAdsEnabled(Profile.getLastUsedRegularProfile());
-        BraveRewardsNativeWorker.getInstance().SetAutoContributeEnabled(true);
```

Comment:
@zenparsing are we setting AC enabled by default ?

Reply:
AC will get set to enabled for first time setup, but will not be re-enabled if the user has already opted-in. This matches the behavior on desktop and is intended to avoid accidental contributions.